### PR TITLE
fix(cli): Enforce a human delay in prompt to fix paste problem

### PR DESCRIPTION
**Pull request by mmastrac**
The permission prompt doesn't wait for quiescent input, so someone pasting a large text file into the console may end up losing the prompt. We enforce a minimum human delay and wait for a 100ms quiescent period before we write and accept prompt input to avoid this problem.

This does require adding a human delay in all prompt tests, but that's pretty straightforward. I rewrote the locked stdout/stderr test while I was in here.

### CODE REVIEW

File: ext/ffi/turbocall.rs [hunk 1]
```diff
@@ -27,6 +27,7 @@ pub(crate) fn is_compatible(sym: &Symbol) -> bool {
       .any(|t| matches!(t, NativeType::Struct(_)))
 }
 
+#[allow(unused)]
```
Comment (mmastrac) [inline ext/ffi/turbocall.rs:30]: Will split this into a separate PR

Comment (bartlomieju) [reply ext/ffi/turbocall.rs]: Woah, why did it become unused now?

**Review by bartlomieju — APPROVED** [review]
